test/test_mps.py

```diff
@@ -9676,6 +9738,18 @@ def get_error_message(self, key, op_name):
             return f"Running test with {op_name} expected to fail due to missing op implementation"
         return f"Running test with {op_name} hangs so skipping"
 
```

Comment:
Also I feel this can be better - we are returning False even when an op is not setup for comparison. Makes it hard to read, imo.

Reply:
Looks more readable to me. Deferring approval to others.